src/MUIDataTable.js

```diff
@@ -425,7 +435,10 @@ class MUIDataTable extends React.Component {
 
   // must be arrow function on local field to refer to the correct instance when passed around
   // assigning it as arrow function in the JSX would cause hard to track re-render errors
-  getTableContentRef = () => this.tableContent.current;
+  getTableContentRef = () => {
+    this.setState({ isPrinting: true });
+    return this.tableRef ? this.tableRef.current : this.tableRef.current;
```

Comment:
This always resolves to this.tableRef.current

Reply:
Yeah, sorry :/